modules/ROOT/nav.adoc

```diff
@@ -7,4 +7,3 @@
 *** xref:heartbeat-vm.adoc[Installing Heartbeat on Server/VM]
 *** xref:pulsar-monitor.adoc[Monitoring Luna using Pulsar Heartbeat]
 *** xref:faqs.adoc[Luna Streaming FAQs]
-** xref:admin-console-VM.adoc[Install Admin Console on Server/VM]
```

Comment:
Why was this removed as part of the PR? Also, should `starlight.adoc` be added to the nav?